src/dpop/DpopClientKeyManager.ts

```diff
@@ -67,7 +67,7 @@ export default class DpopClientKeyManager implements IDpopClientKeyManager {
       await this.storageUtility.set(
         this.getLocalStorageKey(),
         JSON.stringify(jwk),
-        { secure: true }
+        { secure: false }
```

Comment:
I'd add a `TODO` comment here to explain why.

Reply:
Addressed in e65ae11bb5e81297c1fbc3e19e0219f4d3b5282d